WcaOnRails/app/views/panel/index.html.erb

```diff
@@ -83,26 +83,22 @@
     </ul>
   <% end %>
 
-  <% if current_user&.board_member? %>
-    <h2>Board</h2>
-
-    <ul>
-      <li>
-        <%= link_to "List of Senior Delegates", panel_seniors_path %>
-      </li>
-    </ul>
-  <% end %>
-
-  <% if current_user.can_admin_finances? %>
+  <% if current_user.financial_committee? || current_user.board_member? || current_user.admin? %>
```

Comment:
Permissions method in `user` class, please. Does `current_user.can_admin_finances?` fit this use case?

Reply:
I created `can_access_panel` so that I can use the same function when we have to add a panel section in "top navigation right dropdown" which will list all the panels accessible by the user and eventually getting rid of this page.